chart/skywalking/Chart.yaml

```diff
@@ -40,6 +40,6 @@ dependencies:
     condition: postgresql.enabled
   - name: skywalking-banyandb-helm
     alias: banyandb
-    version: 0.0.0-8bdff75
-    repository: oci://ghcr.io/apache/skywalking-banyandb-helm
+    version: 0.2.0
+    repository: oci://registry-1.docker.io/apache
```

Comment:
I think this relies on the official release?

Reply:
Yes, this PR will be ready after the release of BanyanDB Helm 0.2.0.